app/views/doctors/_doctor_search_form.html.haml

```diff
@@ -2,5 +2,4 @@
   = label_tag :q, t("doctor_search_form.label")
   .input-group
     = search_field_tag :q, @q, placeholder: t("doctor_search_form.placeholder")
-    = button_tag t("doctor_search_form.submit_text"), type: "submit", name: nil
-
+    = button_tag t("doctor_search_form.submit_text"), type: "submit", name: nil
```

Comment:
Sorry got confused an commented on this one. Is this meant to be on this commit?